src/models/smartDevices.js

```diff
@@ -1,30 +1,35 @@
 import mongoose from "mongoose"
 
-const smartDeviceSchema = new mongoose.Schema(
+const smartHomeSchema = new mongoose.Schema(
   {
-    _id: {
-      type: mongoose.Schema.Types.ObjectId,
-      required: true,
-      auto: true,
-    },
     homeId: {
       type: String,
       required: true,
+      unique: true,
+    },
+    devices: {
```

Comment:
Devices could already have all 14/17 sensors added with default value false, instead of adding them one by one as the routes come.